testing/kuttl/README.md

```diff
@@ -62,28 +65,28 @@ change those K8s objects before applying them. That means that, if we wanted to
 connection test for PG 13 and PG 14, we would end up writing two nearly identical tests.
 
 Rather than write those multiple tests, we are using `envsubst` to replace some common variables
-in `source` template YAML (the `e2e` and `e2e-other` folders) and writing those files to the
-`testing/kuttl/e2e-generated` and `testing/kuttl/e2e-generated-other` folders.
+and writing those files to the `testing/kuttl/e2e-generated*` directories.
 
 These templated test files can be generated by setting some variables in the command line and
 calling the `make generate-kuttl` target:
 
 ```console
-KUTTL_PG_VERSION=13 KUTTL_POSTGIS_VERSION=3.0 KUTTL_PSQL_IMAGE=registry.developers.crunchydata.com/crunchydata/crunchy-postgres:centos8-13.5-0 make generate-kuttl
+KUTTL_PG_VERSION=13 KUTTL_POSTGIS_VERSION=3.0 make generate-kuttl
```

Comment:
I like leaving the KUTTL_PSQL_IMAGE in the example here which shows how you can easily set an image to use when running via the command line.

Reply:
It's set the same way as these other variables. We shouldn't need to change it when testing different versions.

Do you think we should put every variable in this readme, or leave it to the target for now?

https://github.com/CrunchyData/postgres-operator/blob/2b2e019e117ab1ce80cf16cf138e22e1039c14ad/Makefile#L236-L240